### ISSUE TITLE
Could you please upload a processing_file.qs example ?

### ISSUE THREAD

**Issue by ygbingo** [Noel-Research-Group/Robochem_v1.0#1]
### Description:
I downloaded the source code and followed the instructions to execute the command "streamlit run .\🏡_Home.py" locally. After opening it on the web and filling in each step's parameters as instructed, upon clicking "RUN Platform," I received the message: "Processing file not added to experiment folder. Please add this folder named 'processing_file.qs' before running the experiments. Excel file containing the vial positions used by the Liquid Handler has not been generated." It seems like there might be a missing file in the project. Could you provide an example of the 'qs' file?

### My Environment
System: Win11
build env: from README

### To Whom It May Concern,

I hope this message finds you well. I recently encountered an issue while running the Streamlit application. I downloaded the source code and diligently followed the provided instructions to set up and run the application locally. However, upon reaching the stage of executing the command ```streamlit run .\🏡_Home.py``` and interacting with the web interface, I encountered an error message.

The error message states: **"Processing file not added to experiment folder. Please add this folder named 'processing_file.qs'** before running the experiments. Excel file containing the vial positions used by the Liquid Handler has not been generated."

Based on this error message, it seems that there might be a missing file or directory in the project structure. I have checked the contents of the downloaded source code, but I couldn't find any reference to the mentioned file or folder. Could you please provide guidance on what the 'processing_file.qs' should contain and where it should be located within the project directory?

Additionally, it would be immensely helpful if you could provide an example of the 'qs' file to ensure that I am including the correct information in the right format.

Thank you very much for your assistance in resolving this issue. I look forward to your prompt response.

**Comment by aslatte** [comment]
Hi there,

The .qs file is a scripting file for MestReNova, an example file can be found in Robochem>Platform_>NMR_Control_Loop>processing_files>processing_file_example.qs. This should be added to the experiment folder that's automatically generated by the GUI and renamed 'processing_file.qs'. This should be located C:\\Users\\**user**\\Robochem_Data\\**user_name**\\**experiment_ID**\\. the **bold** parts will be unique to you.

If you want to get it to process NMRs for you, you'll have to make also a .mnp file in MestReNova which is the MestReNova processing template to be used on your NMRs. The scripting file will have to be updated and given your .mnp file and location replacing the line with the location and name of the .mnp file with the .qs you created on your own system
from processing_file_example.qs
'procFileName = "C:/Users/Platform/code/RoboChem_auto-optimization-platform/Platform_/NMR_control_loop/processing_files/example.mnp";' **->** procFileName = "your_location/your_mnp.mnp";' 

Please let me know if this helps.

**Comment by ygbingo** [comment]
Hello,

I hope this message finds you well. Firstly, I would like to extend my gratitude for your prompt and helpful response. I have followed your instructions and have successfully re-run the platform. After clicking the **"Generate Excel"** button, I was able to locate the experimental folder and the generated Excel file in my computer's user directory at _C:\Users\<>\Platform_Data\<>\_.

Following your guidance, I have placed the processing_file.qs in the aforementioned directory. To my delight, the "**Run Platform**" button is now operational on the Run Platform interface!
